hrepr/style/hrepr.css

```
@@ -34,6 +34,24 @@
 	color: #888;
 }
 
+.hrepr-bytes {
+	color: #666;
+    font-family: monospace;
```

You're mixing tabs and spaces here.

```suggestion
	font-family: monospace;
```

(Also, I had never noticed GitHub set a tab size of 6, that's pretty clever.)